openpype/hosts/nuke/api/workfile_template_builder.py

```diff
@@ -33,6 +34,30 @@
 class NukeTemplateBuilder(AbstractTemplateBuilder):
     """Concrete implementation of AbstractTemplateBuilder for nuke"""
 
+    def open_template(self):
+        """Open template in current scene.
+
+        Args:
+            path (str): A path to current template (usually given by
+            get_template_preset implementation)
+        """
+
+        template_preset = self.get_template_preset()
+        template_path = template_preset["path"]
+
+        if not os.path.exists(template_path):
+            nuke.message("Template doesn't exist: {}".format(template_path))
+            return
+
+        result = nuke.ask(
+            "This will replace current scene with template. Continue?"
+        )
+        if not result:
+            return
+
+        print("opening template {}".format(template_path))
+        nuke.scriptOpen(template_path)
```

Comment:
This does not clear the current scene. Maybe use [`open_file`](https://github.com/ynput/OpenPype/blob/bebaeee66a626196f4cb99df4728a7fa3f5b8010/openpype/hosts/nuke/api/workio.py#L22-L32)

Reply:
Do you think we should do the same for Maya?

Reply:
Yeah, actually you could do this in `pipeline/workfile/workfile_template_builder.py` and use Qt for showing the dialogs, then its cross host and less code.